package.json

```diff
@@ -78,21 +78,23 @@
     "@types/d3-format": "^3.0.1",
     "@types/d3-time-format": "^4.0.0",
     "@types/lodash": "^4.14.180",
-    "@types/node": "^17.0.23",
-    "@types/react": "^17.0.43",
+    "@types/node": "18.7.18",
+    "@types/react": "18.0.20",
+    "@types/react-dom": "^18.0.6",
     "autoprefixer": "^10.4.8",
     "bootstrap": "4.6.0",
-    "cypress": "^10.1.0",
+    "cypress": "10.8.0",
     "eslint": "^7.32.0",
     "eslint-config-next": "^12.1.0",
     "file-loader": "^6.2.0",
-    "mdi-react": "^8.2.0",
+    "mdi-react": "9.0.0",
     "next-sitemap": "^3.1.22",
     "postcss": "^8.4.14",
     "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-preset-env": "^7.7.2",
     "prettier": "^2.5.1",
     "react-bootstrap": "2.0.0",
+    "react-icons": "^4.4.0",
```

Comment:
Curious what you think about this package, but I think I found it as a sweet solution to allow us to reference any icon under the sun. 😄  Gives us flexibility to reference missing icons in mdi-react, as well as allowing multi imports on one line from individual icon libraries. [Their docs here](https://react-icons.github.io/react-icons).

Reply:
More icon options & importing on 1 line are big plus's! Does this have a negative impact on our bundle size?

Reply:
No negative impact, which is great because we aren't importing every icon and it supports tree shaking. Unlike MUI which was [huge](https://github.com/sourcegraph/about/issues/5432#issuecomment-1155667416)!

I added Next.js' Bundle Analyzer ([bd1ed5e](https://github.com/sourcegraph/about/pull/5743/commits/bd1ed5ec03019453471d9c0eec4b56c7cb719c07)) to our scripts so we can test and analyze at anytime via `yarn analyze`. In my screenshots below, you can see how tiny adding the 1 Discord Icon is in this PR: 1.43 KB. 🤓 It will add up overtime of course as we add more icons, but at least we're not importing every icon all at once (as seen [here](https://github.com/sourcegraph/about/pull/5476/files#diff-f8824e0763ada0a69090743c639d25552e49153e91fd675e481d7f610b4a5b44R4) 😬).

![Screen Shot 2022-09-19 at 3 28 40 PM](https://user-images.githubusercontent.com/1733936/191100750-28216aac-0e74-4df9-a342-3259ed7fb3aa.png)
![Screen Shot 2022-09-19 at 3 29 18 PM](https://user-images.githubusercontent.com/1733936/191100756-a1305c0f-f098-4468-b451-c8858693f40c.png)

Reply:
Woohoo awesome! Oh yea we still need to finish the styled icons at some point 😅 -- Dynamic imports will be interesting, I haven't done that before. This looks great - thanks for the analyzer script! Could be a good thing to add in our README too, alongside the testing scripts 🤷🏻